ACM_General/events/models.py

```diff
@@ -47,6 +47,46 @@ def get_path_for_flier(instance, filename):
     )
 
 
+class Tag(models.Model):
+    """
+    Class used for adding classification tags on events.
+    """
+
+    #: The name of the tag that is displayed on the event creation form; represented
```

Comment:
This will not exclusively be what it will be used for. It will be used anytime that we display the Tag. Rephrase to make more generic.